log/sequencer_test.go

```diff
@@ -165,23 +163,28 @@ var (
 		Revision:       6,
 		TreeSize:       22,
 	}
-	updatedSignedRoot21, _ = tcrypto.NewSigner(fixedGoSigner, crypto.SHA256).SignLogRoot(updatedRoot21)
+	updatedSignedRoot21 = makeSLR(updatedRoot21)
 
 	emptyRoot = &types.LogRootV1{
 		TimestampNanos: uint64(fakeTime.Add(-10 * time.Millisecond).UnixNano()),
 		TreeSize:       0,
 		Revision:       2,
 		RootHash:       rfc6962.DefaultHasher.EmptyRoot(),
 	}
-	signedEmptyRoot, _        = tcrypto.NewSigner(fixedGoSigner, crypto.SHA256).SignLogRoot(emptyRoot)
-	updatedSignedEmptyRoot, _ = tcrypto.NewSigner(fixedGoSigner, crypto.SHA256).SignLogRoot(&types.LogRootV1{
+	signedEmptyRoot        = makeSLR(emptyRoot)
+	updatedSignedEmptyRoot = makeSLR(&types.LogRootV1{
 		TimestampNanos: uint64(fakeTime.UnixNano()),
 		TreeSize:       0,
 		Revision:       3,
 		RootHash:       rfc6962.DefaultHasher.EmptyRoot(),
 	})
 )
 
+func makeSLR(root *types.LogRootV1) *trillian.SignedLogRoot {
+	logRoot, _ := root.MarshalBinary()
```

Comment:
This is used when creating global variables, so that's not possible. I somewhat mimicked the previous behaviour with this, which would throw away the error from `SignLogRoot` as well...